CHANGELOG.md

```diff
@@ -31,6 +31,8 @@
 - Fix location issue in error messages with JSX V4 where the multiple props types are defined https://github.com/rescript-lang/syntax/pull/655
 - Fix location issue in make function in JSX V4 that breaks dead code elimination https://github.com/rescript-lang/syntax/pull/660
 - Fix parsing (hence pretty printing) of expressions with underscore `_` and comments.
+- Fix printing of comments inside JSX tag https://github.com/rescript-lang/syntax/pull/664
```

Comment:
Oh this must have been missed somewhere.

Reply:
This change is also included in this PR. See details in the PR description

Reply:
OK thanks.